app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java

```diff
@@ -68,8 +68,9 @@ public void init(final int position, final Contribution contribution) {
     this.contribution = contribution;
     this.position = position;
     titleView.setText(contribution.getMedia().getMostRelevantCaption());
```

Comment:
If `contribution.getMedia()` is null then this line fails first, I guess?
More importantly, what are the situations where a null `contribution.getMedia()` makes sense? If there is no situation where this makes sense, better search for the root cause.
Thanks! :-)

Reply:
@nicolas-raoul I don't know the root cause. Unfortunately, I don't familiar with the API of wikidata. Should I hot fix this for now like  this
```java
String imageSource = "";
    if (contribution.getMedia() != null) {
      titleView.setText(contribution.getMedia().getMostRelevantCaption());
      authorView.setText(contribution.getMedia().getCreator());
      imageSource = chooseImageSource(contribution.getMedia().getThumbUrl(),
        contribution.getLocalUri());
    }
```

Or Should I  familiar with the API?
P.S I can't find a communication channel. Is there a communication channel ? Thanks!